src/stan/mcmc/hmc/hamiltonians/diag_e_point.hpp

```diff
@@ -53,6 +53,7 @@ namespace stan {
        *
        * @param writer Stan writer callback
        */
+      inline
```

Comment:
We're trying to pack as much onto a single line as we can these days, but it's a minor issue, so I'm not going to hold this important request up for minor formatting.